roles/edpm_nova/tasks/configure.yml

```diff
@@ -16,7 +16,6 @@
         - nova
 
     - name: Assert edpm_nova_extra_bind_mounts source directories exist
-      become: true
```

Comment:
the become here was becasue these directores may not be owned by the ansible user.

however if we are just checking it exists then that is probably ok.

Reply:
This is assert on results of previous task. Become is completely useless here.

Reply:
oh yes your correct i didnt fully expand the context.